system-int-test/test/cmr/system_int_test/search/facets/collection_facets_v2_search_test.clj

```diff
@@ -655,57 +656,57 @@
     ;; Variable1 is associated with coll1 and coll2
     ;; Variable2 is associated with coll2 and coll3
     ;; SomeVariable is associated with coll4
-    (variable-util/associate-by-concept-ids token
-                                            variable1-concept-id
-                                            [{:concept-id (:concept-id coll1)}
-                                             {:concept-id (:concept-id coll2)}])
-    (variable-util/associate-by-concept-ids token
-                                            variable2-concept-id
-                                            [{:concept-id (:concept-id coll2)}
-                                             {:concept-id (:concept-id coll3)}])
-    (variable-util/associate-by-concept-ids token
-                                            variable3-concept-id
-                                            [{:concept-id (:concept-id coll4)}])
+    (au/associate-by-concept-ids token
+                                 variable1-concept-id
+                                 [{:concept-id (:concept-id coll1)}
+                                  {:concept-id (:concept-id coll2)}])
+    (au/associate-by-concept-ids token
+                                 variable2-concept-id
+                                 [{:concept-id (:concept-id coll2)}
+                                  {:concept-id (:concept-id coll3)}])
+    (au/associate-by-concept-ids token
+                                 variable3-concept-id
+                                 [{:concept-id (:concept-id coll4)}])
     (index/wait-until-indexed)
     (testing "variable facets are disabled by default")
+    (let [facets-result (search-and-return-v2-facets
+                         {:variables-h {:0 {:variable "Variable1"}}})]
+      (assert-facet-field-not-exist facets-result "Measurements" "Measurement1")
+      (assert-facet-field-not-exist facets-result "Measurements" "Measurement2"))
+    (testing "variable facets enabled")
+    (dev-sys-util/eval-in-dev-sys
```

Comment:
Indentation is off - these should be within the variable facets enabled test section.

Reply:
Good catch. Done.